Clients/GUI/Shared/NewsComponent.razor

```diff
@@ -0,0 +1,66 @@
+﻿@page  "/NewsComponent"
```

Comment:
I don't think we need a special address for it

Reply:
This is not address, this is the name of component